### DISCUSSION TITLE
Feature to export messages from a specific user in a guild

### DISCUSSION THREAD

**Discussion by fluffydino** [Tyrrrz/DiscordChatExporter#566] [General]
I think it would be a great option of you could export messages from a specific person in a guild. Personally, I need a record of all my messages, and to export the entire server would take weeks and terabytes of storage.

**Comment by 96-LB** [comment]
I think this falls under #160. As this is one of the most requested features, I'm willing to start looking into it if nothing is currently in the works.

**Reply by Tyrrrz** [reply]
That's correct @96-LB. Let's start with a discussion on how to approach this. Ideally by finding something more or less already fit for the purpose, to avoid building our own query language.
